tools/generate_images.js

```diff
@@ -81,7 +81,7 @@ const parseArgs = () => {
   });
 
   backends = backends ?? {
-    pptr: true,
+    jsdom: true,
```

Comment:
Maybe instead of deleting the pptr line, we can comment it out? These scripts are not well documented so it might be an easy way to show which flags are available.

Reply:
The options are in the code a few lines above

https://github.com/vexflow/vexflow/blob/7f871dcc05856de626ec91bf8d29d590eb6dda28/tools/generate_images.js#L53-L58

also in the help

https://github.com/vexflow/vexflow/blob/7f871dcc05856de626ec91bf8d29d590eb6dda28/tools/generate_images.js#L13-L21

Reply:
@ronyeh is the above fine or do you want me to add a comment?